.readthedocs.yml

```
@@ -7,19 +7,21 @@ build:
   os: ubuntu-22.04
   tools:
     python: mambaforge-latest
```

```suggestion
    python: miniforge-latest
```

mambaforge is deprecated and miniforge is identical to it now. I think this will work.